gamemodes/terrortown/entities/entities/ttt_c4/shared.lua

```diff
@@ -476,29 +474,10 @@ end
 -- Timer configuration handling
 
 if SERVER then
-	---
-	-- Inform traitors about us
-	-- @param boolean armed
-	-- @realm server
-	function ENT:SendWarn(armed)
-		net.Start("TTT_C4Warn")
```

Comment:
Remove this from sv_main
Also remove the net receive in cl_init

We could discuss if we keep this for compatibility but i do not think anyone is abusing this for displaying their own bombs in the radar currently

Reply:
are you sure? aren't addons such as the turtlenade using this?

Reply:
Im not sure, thats why we should discuss this

Reply:
as far as I can tell other addons add their own custom syncing, but use the old TTT rendering: 
![image](https://github.com/TTT-2/TTT2/assets/13639408/715b6bb8-485c-4475-8afe-add6ebed3e59)